test/bigtest/tests/import-jobs-test.js

```diff
@@ -45,14 +123,25 @@ describe('Import files with previous draft jobs', () => {
     expect(returnToAssignJobs.title).to.be.equal(translation['returnToAssign.message']);
   });
 
-  describe('resets to initial state', () => {
+  describe('delete button', () => {
     beforeEach(async () => {
-      await returnToAssignJobs.deleteFileExtension();
+      await returnToAssignJobs.deleteButtonClick();
+      await returnToAssignJobs.deleteButtonClick();
     });
 
-    it('renders', () => {
+    it('resets to initial state', () => {
       expect(importJobs.isPresent).to.be.true;
       expect(returnToAssignJobs.isPresent).to.be.false;
     });
   });
+
+  describe('resume button', () => {
+    beforeEach(async () => {
+      await returnToAssignJobs.resumeButtonClick();
+    });
+
+    it('navigates to ChooseJobsProfile page', () => {
```

Comment:
`navigates to "Choose jobs profile" page`. The idea is to get rid of variable names. Handle similar cases.